docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  pg:
+    image: postgres:10.0-alpine
+    ports:
+      - 5432:5432
+    volumes:
+      - pg_data:/var/lib/postgresql/data/pg_data
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql

```

Comment:
It is initialized when the tng-vnv-platform-adapter starts 
Check here:
https://github.com/sonata-nfv/tng-vnv-platform-adapter/blob/master/main.py#L602

Reply:
schema is initialized but there are no data